### PR TITLE
Improve grammar, fix typos

### PR DESCRIPTION
**Pull request by AtilioA**
#### What is the purpose of this change?
This fixes multiple instances of typos in the documentation, e.g. when "third party", "open source", "feature rich" should actually be "third-party", "open-source", "feature-rich", respectively, when there are missing commas, etc.

#### Was the change discussed in an issue or in the forum before?
To some extent, as in #5332 and #214.

#### Checklist

- [x] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-pull-request).
- [ ] I have added tests for all changes in this PR if appropriate.
- [ ] I have added documentation for the changes if appropriate.
- [x] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [x] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

**Comment by ncw** [comment]
Thank you for these fixes - they look very useful.

 I note it has some auto generated files in it which we might want to drop - eg MANUAL.* rclone.1 - these get updated at the point of release.

@albertony - would you like to review this PR?

**Comment by albertony** [comment]
> would you like to review this PR?

I can read through it, yes. But if there are a lot of spelling nitpicks where there is doubt, I will not be able have a strong opinion.. 🦺

File: MANUAL.html [hunk 1]
```diff
@@ -31,22 +31,22 @@ <h1 id="rclone-syncs-your-files-to-cloud-storage">Rclone syncs your files to clo
 <li><a href="https://rclone.org/donate/">Donate.</a></li>
 </ul>
 <h2 id="about">About rclone</h2>
-<p>Rclone is a command line program to manage files on cloud storage. It is a feature rich alternative to cloud vendors' web storage interfaces. <a href="#providers">Over 40 cloud storage products</a> support rclone including S3 object stores, business &amp; consumer file storage services, as well as standard transfer protocols.</p>
+<p>Rclone is a command-line program to manage files on cloud storage. It is a feature-rich alternative to cloud vendors' web storage interfaces. <a href="#providers">Over 40 cloud storage products</a> support rclone including S3 object stores, business &amp; consumer file storage services, as well as standard transfer protocols.</p>
```
Comment (albertony) [inline MANUAL.html:34]: This file is auto-generated from many markdown files, and comment-based documentation in source. So the changes must be done in the correct sources, and this file removed from the PR.
This specific line has been fixed in correct source, docs/content/_index.md, but I haven't gone through to verify if this is the case for all of the changes.

File: MANUAL.md [hunk 2]
```diff
@@ -16,8 +16,8 @@
 
 ## About rclone {#about}
 
-Rclone is a command line program to manage files on cloud storage. It
-is a feature rich alternative to cloud vendors' web storage
+Rclone is a command-line program to manage files on cloud storage. It
```
Comment (albertony) [inline MANUAL.md:19]: This is also auto-generated file.

File: MANUAL.txt [hunk 3]
```diff
@@ -18,8 +18,8 @@ RCLONE SYNCS YOUR FILES TO CLOUD STORAGE
 
 About rclone
 
-Rclone is a command line program to manage files on cloud storage. It is
```
Comment (albertony) [inline MANUAL.txt:21]: This is also auto-generated file.

File: docs/content/commands/rclone.md [hunk 4]
```diff
@@ -36,7 +36,7 @@ See the [global flags page](/flags/) for global options not listed here.
 
 * [rclone about](/commands/rclone_about/)	 - Get quota information from the remote.
 * [rclone authorize](/commands/rclone_authorize/)	 - Remote authorization.
-* [rclone backend](/commands/rclone_backend/)	 - Run a backend specific command.
+* [rclone backend](/commands/rclone_backend/)	 - Run a backend-specific command.
```
Comment (albertony) [inline docs/content/commands/rclone.md:39]: This is also autogenated file, see line 6:

> \# autogenerated - DO NOT EDIT, instead edit the source code in cmd/ and as part of making a release run "make commanddocs"

File: docs/content/commands/rclone_serve_restic.md [hunk 5]
```diff
@@ -15,7 +15,7 @@ rclone serve restic implements restic's REST backend API
 over HTTP.  This allows restic to use rclone as a data storage
 mechanism for cloud providers that restic does not support directly.
 
-[Restic](https://restic.net/) is a command line program for doing
```
Comment (albertony) [inline docs/content/commands/rclone_serve_restic.md:18]: Autogenerated file.

File: docs/content/commands/rclone_obscure.md [hunk 6]
```diff
@@ -11,7 +11,7 @@ Obscure password for use in the rclone config file.
 
 ## Synopsis
 
-In the rclone config file, human readable passwords are
+In the rclone config file, human-readable passwords are
```
Comment (albertony) [inline docs/content/commands/rclone_obscure.md:14]: Autogenerated file.

File: docs/content/commands/rclone_mount.md [hunk 7]
```diff
@@ -68,7 +68,7 @@ or newer on some platforms depending on the underlying FUSE library in use.
 To run rclone mount on Windows, you will need to
 download and install [WinFsp](http://www.secfs.net/winfsp/).
 
-[WinFsp](https://github.com/billziss-gh/winfsp) is an open source
```
Comment (albertony) [inline docs/content/commands/rclone_mount.md:71]: Autogenerated file.

File: docs/content/commands/rclone_lsl.md [hunk 8]
```diff
@@ -34,17 +34,17 @@ There are several related list commands
   * `lsf` to list objects and directories in easy to parse format
   * `lsjson` to list objects and directories in JSON format
 
-`ls`,`lsl`,`lsd` are designed to be human readable.
```
Comment (albertony) [inline docs/content/commands/rclone_lsl.md:37]: Autogenerated file.

File: docs/content/commands/rclone_lsjson.md [hunk 9]
```diff
@@ -60,7 +60,7 @@ If "remote:path" contains the file "subfolder/file.txt", the Path for "file.txt"
 will be "subfolder/file.txt", not "remote:path/subfolder/file.txt".
```
Comment (albertony) [inline docs/content/commands/rclone_lsjson.md:60]: Autogenerated file.

File: docs/content/commands/rclone_lsf.md [hunk 10]
```diff
@@ -59,13 +59,13 @@ can be returned as an empty string if it isn't available on the object
 the object and "UNSUPPORTED" if that object does not support that hash
```
Comment (albertony) [inline docs/content/commands/rclone_lsf.md:59]: Autogenerated file.

File: docs/content/commands/rclone_lsd.md [hunk 11]
```diff
@@ -44,17 +44,17 @@ There are several related list commands
   * `lsf` to list objects and directories in easy to parse format
```
Comment (albertony) [inline docs/content/commands/rclone_lsd.md:44]: Autogenerated file.

File: docs/content/commands/rclone_ls.md [hunk 12]
```diff
@@ -34,17 +34,17 @@ There are several related list commands
   * `lsf` to list objects and directories in easy to parse format
```
Comment (albertony) [inline docs/content/commands/rclone_ls.md:34]: Autogenerated file.

File: docs/content/commands/rclone_dedupe.md [hunk 13]
```diff
@@ -32,7 +32,7 @@ name.  It will do this iteratively until all the identically named
 directories have been merged.
```
Comment (albertony) [inline docs/content/commands/rclone_dedupe.md:32]: Autogenerated file.

File: docs/content/commands/rclone_config_update.md [hunk 14]
```diff
@@ -15,7 +15,7 @@ Update options in an existing remote.
 Update an existing remote's options. The options should be passed in
```
Comment (albertony) [inline docs/content/commands/rclone_config_update.md:15]: Autogenerated file.

File: docs/content/commands/rclone_config_password.md [hunk 15]
```diff
@@ -15,7 +15,7 @@ Update password in an existing remote.
 Update an existing remote's password. The password
```
Comment (albertony) [inline docs/content/commands/rclone_config_password.md:15]: Autogenerated file.

File: docs/content/commands/rclone_config_create.md [hunk 16]
```diff
@@ -15,7 +15,7 @@ Create a new remote with name, type and options.
 Create a new remote of `name` with `type` and options.  The options
```
Comment (albertony) [inline docs/content/commands/rclone_config_create.md:15]: Autogenerated file.

File: docs/content/commands/rclone_check.md [hunk 17]
```diff
@@ -13,7 +13,7 @@ Checks the files in the source and destination match.
```
Comment (albertony) [inline docs/content/commands/rclone_check.md:13]: Autogenerated file.

File: docs/content/commands/rclone_about.md [hunk 18]
```diff
@@ -43,7 +43,7 @@ Applying a `--full` flag to the command prints the bytes in full, e.g.
     Trashed: 104857602
     Other:   8849156022
 
-A `--json`flag generates conveniently computer readable output, e.g.
```
Comment (albertony) [inline docs/content/commands/rclone_about.md:46]: Autogenerated file.

File: docs/content/commands/rclone_backend.md [hunk 19]
```diff
@@ -1,18 +1,18 @@
...
 title: "rclone backend"
-description: "Run a backend specific command."
+description: "Run a backend-specific command."
 slug: rclone_backend
 url: /commands/rclone_backend/
 # autogenerated - DO NOT EDIT, instead edit the source code in cmd/backend/ and as part of making a release run "make commanddocs"
 ---
 # rclone backend
 
-Run a backend specific command.
```
Comment (albertony) [inline docs/content/commands/rclone_backend.md:10]: Autogenerated file.

File: docs/content/commands/rclone_tree.md [hunk 20]
```diff
@@ -25,7 +25,7 @@ For example
     └── subdir
         ├── file4
         └── file5
-
```
Comment (albertony) [inline docs/content/commands/rclone_tree.md:28]: Autogenerated file.

File: docs/content/drive.md [hunk 21]
```diff
@@ -608,7 +608,7 @@ a non root folder as its starting point.
 
 #### --drive-service-account-file
 
-Service Account Credentials JSON file path
```
Comment (albertony) [inline docs/content/drive.md:611]: Autogenerated section of file.  See line 544:

> {{< rem autogenerated options start" - DO NOT EDIT - instead edit fs.RegInfo in backend/drive/drive.go then run make backenddocs" >}}

File: docs/content/googlephotos.md [hunk 22]
```diff
@@ -414,7 +414,7 @@ Year limits the photos to be downloaded to those which are uploaded after the gi
 
 Also view and download archived media.
 
-By default rclone does not request archived media. Thus, when syncing,
+By default, rclone does not request archived media. Thus, when syncing,
```
Comment (albertony) [inline docs/content/googlephotos.md:417]: Autogenerated section.

File: docs/content/http.md [hunk 23]
```diff
@@ -131,15 +131,15 @@ Use this to set additional HTTP headers for all transactions
...
+For example, to set a Cookie use 'Cookie,name=value', or '"Cookie","name=value"'.
 
 You can set multiple headers, e.g. '"Cookie","name=value","Authorization","xxx"'.
 
 
 - Config:      headers
 - Env Var:     RCLONE_HTTP_HEADERS
 - Type:        CommaSepList
-- Default:     
+- Default:
```
Comment (albertony) [inline docs/content/http.md:142]: Autogenerated section.

File: docs/content/onedrive.md [hunk 24]
```diff
@@ -301,7 +301,7 @@ The type of the drive ( personal | business | documentLibrary )
 
 Set to make OneNote files show up in directory listings.
 
-By default rclone will hide OneNote files in directory listings because
+By default, rclone will hide OneNote files in directory listings because
```
Comment (albertony) [inline docs/content/onedrive.md:304]: Autogenerated section.

File: docs/content/rc.md [hunk 25]
```diff
@@ -651,7 +651,7 @@ If the rate parameter is not supplied then the bandwidth is queried
 The format of the parameter is exactly the same as passed to --bwlimit
 except only one bandwidth may be specified.
 
-In either case "rate" is returned as a human readable string, and
+In either case "rate" is returned as a human-readable string, and
```
Comment (albertony) [inline docs/content/rc.md:654]: Autogenerated section.

File: docs/content/sftp.md [hunk 26]
```diff
@@ -346,7 +346,7 @@ when the ssh-agent contains many keys.
 
 #### --sftp-use-insecure-cipher
 
-Enable the use of insecure ciphers and key exchange methods. 
+Enable the use of insecure ciphers and key exchange methods.
```
Comment (albertony) [inline docs/content/sftp.md:349]: Autogenerated section.

File: docs/content/webdav.md [hunk 27]
```diff
@@ -207,15 +207,15 @@ Use this to set additional HTTP headers for all transactions
...
+For example, to set a Cookie use 'Cookie,name=value', or '"Cookie","name=value"'.
 
 You can set multiple headers, e.g. '"Cookie","name=value","Authorization","xxx"'.
 
 
 - Config:      headers
 - Env Var:     RCLONE_WEBDAV_HEADERS
 - Type:        CommaSepList
-- Default:     
+- Default:
```
Comment (albertony) [inline docs/content/webdav.md:218]: Autogenerated section.

File: rclone.1 [hunk 28]
```diff
@@ -21,8 +21,8 @@ Install (https://rclone.org/install/)
 Donate. (https://rclone.org/donate/)
 .SS About rclone
 .PP
-Rclone is a command line program to manage files on cloud storage.
```
Comment (albertony) [inline rclone.1:24]: Autogenerated file.

**Review by albertony — COMMENTED** [review]
Great work 😅 There are quite a few changes in autogenerated files and autogenerated sections of files that should be cleaned from the pr, added inline comments to mark the ones I found, but other than that it looks ⭐

**Comment by ivandeex** [comment]
@AtilioA 
Please fix merge conflicts

**Comment by AtilioA** [comment]
Sorry, I lost track of this. I'll work on it soon.

**Comment by ivandeex** [comment]
Thank you!

**Comment by AtilioA** [comment]
IIRC, I fixed it and was waiting for a review. In any case, I should be able to go over this again in a week or two.

**Comment by albertony** [comment]
> Unfortunately this can't be merged or even reviewed due to merge conflicts.
> @albertony could you help here?

Sure; just did. Did a plain merge from latest upstream/master and resolved the conflicts. Hope I did nothing wrong, haven't messed around with other people's PR branches before. 😨

**Comment by albertony** [comment]
Rebased on master and fixed conflicts.

**Review by ncw — APPROVED** [review]
I've been through this and it looks good to me - let's merge :-)

@albertony do you want to do the honours?

(There is still one spacing change in auto generated section but I suggest we ignore that as it is minor and will get put back most likely!)

**Comment by albertony** [comment]
Merged.

@AtilioA: Thank you for your contribution!

**Pull request merged**